src/long.ts

```diff
@@ -186,6 +186,16 @@ export class Long {
     return Long.fromBits(value % TWO_PWR_32_DBL | 0, (value / TWO_PWR_32_DBL) | 0, unsigned);
   }
 
+  /**
+   * Returns a Long representing the given value, provided that it is a finite number. Otherwise, zero is returned.
+   * @param value - The number in question
+   * @param unsigned - Whether unsigned or not, defaults to signed
+   * @returns The corresponding Long value
+   */
+  static fromBigInt(value: bigint, unsigned?: boolean): Long {
+    return Long.fromString(value.toString(10), unsigned);
+  }
```

Comment:
While we leave the serializer in a state where a translation must be made manually I figure it would still be good to provide these `to` and `from` helpers, thoughts?